File: programme/area-of-right-angle-triangle/areaofrightangletriangle.go
// Write a programme to find area of right angle triangle

package main

import (
	"fmt"
	"math"
)

func areaOfRightAngleTriangle(base, height float64) float64 {
	return 0.5 * base * height
}

func roundFloat(val float64, precision uint) float64 {
	var ratio float64 = math.Pow(10, float64(precision))
	return math.Round(val*ratio) / ratio
}

func main() {
	var base, height, area float64

	fmt.Print("base : ")
	fmt.Scan(&base)

	fmt.Print("height : ")
	fmt.Scan(&height)

	area = areaOfRightAngleTriangle(base, height)
	fmt.Print("\narea : ", roundFloat(area, 2))
}

File: programme/area-of-right-angle-triangle/area-of-right-angle-triangle.cs
// Write a programme to find area of right angle triangle
using System;

public class AreaOfRightAngleTriangle
{
    public static void Main()
    {
        Console.Write("base   : ");
        int triangleBase = int.Parse(Console.ReadLine());
        Console.Write("height : ");
        int triangleHeight = int.Parse(Console.ReadLine());

        int area = CalculateArea(triangleBase, triangleHeight);
        Console.WriteLine("\nArea : " + area);
    }

    public static int CalculateArea(int triangleBase, int triangleHeight)
    {
        var area = 0.5 * triangleBase * triangleHeight;
        return (int) area;
    }
}
